clients/ts/FunctionalTests/ts/index.ts

```diff
@@ -4,6 +4,7 @@
 console.log("SignalR Functional Tests Loaded");
 
 import "es6-promise/dist/es6-promise.auto.js";
+import "./ConnectionTests";
```

Comment:
So... uh.. when I brought the tests back I forgot to bring back the import. These haven't been running since the IConnection refactorings in TypeScript. They all still pass though, so no worries.